source/declarations.tex

```diff
@@ -479,9 +479,11 @@
 \end{example}
 
 \pnum
-The \tcode{mutable} specifier shall appear only in the declaration of
+The \keyword{mutable} specifier shall appear only in the declaration of
 a non-static data member\iref{class.mem}
 whose type is neither const-qualified nor a reference type.
+A member subobject\iref{intro.object} corresponding to a non-static
+data member declared with the \keyword{mutable} specifier is a \defnadj{mutable}{subobject}.
```

Comment:
Should "mutable subobject" be transitive? That is, should a (non-const) subobject of a mutable subobject also be a mutable subobject? The use of "mutable subobject" in [basic.type.qualifier]/1.1 in the definition of "const object" appears to assume transitivity -- but maybe it means "or a non-mutable <ins>direct</ins> subobject of a const object"